src/components/About/AppDetail.tsx

```diff
@@ -91,6 +106,7 @@ const AppDetail: React.FC<TableContainerProps> = ({ ...otherProps }) => {
               <Link href='mailto:amwebexpert@gmail.com'>André Masson</Link>
             </StyledTableCell>
           </StyledTableRow>
+          {buildContributors()}
```

Comment:
```suggestion
          <Contributors />
```

Note: you can create `Contributors.tsx` just beside `AppDetail.tsx`